index.html

```diff
@@ -10,6 +10,21 @@
 <body>
   <header>
     <a href="#" id="my-logo">Farida's Logo</a>
+    <img src="images/home.png" alt="description" id="home"/>
   </header>
```

Comment:
- [ ] The hamburger button on the right should wrap with a `nav` tag. Kindly fix it :+1: 
- [ ] The content should be aligned with flexbox. Please work on that. :+1: